wrappers/python/pyrs_types.cpp

```diff
@@ -7,6 +7,30 @@ Copyright(c) 2017 Intel Corporation. All Rights Reserved. */
 void init_types(py::module &m) {
     /** rs2_types.hpp **/
     // error+subclasses
+    enum product_line
```

Comment:
Why is this enum needed? You can just put the values in each line on lines 24-32...